cloudformation.yaml

```diff
@@ -105,6 +105,6 @@ Resources:
                 ScheduleLambda:
                     Type: Schedule
                     Properties:
-                        Schedule: cron(12 14 * * ? *) # run daily at 14:12
+                        Schedule: cron(12 14 * * ? 2031) # run daily at 14:12
```

Comment:
I assume lambda schedule's support years in cron format 👍

Reply:
According to [the docs](https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents-expressions.html) cron accepts a year value